evalml/pipelines/component_graph.py

```diff
@@ -205,36 +205,40 @@ def _fit_transform_features_helper(self, needs_fitting, X, y=None):
         component_outputs = self._compute_features(
             self.compute_order[:-1], X, y=y, fit=needs_fitting
         )
-        final_component_inputs = []
-
-        parent_inputs = [
-            parent_input
-            for parent_input in self.get_inputs(self.compute_order[-1])
-            if parent_input[-2:] != ".y"
-        ]
-        for parent in parent_inputs:
-            parent_output = component_outputs.get(
-                parent, component_outputs.get(f"{parent}.x")
-            )
-            if isinstance(parent_output, pd.Series):
-                parent_output = pd.DataFrame(parent_output, columns=[parent])
```

Comment:
Do we no longer need to change the series to a dataframe?

Reply:
I think no, this might have been outdated--we use ww's `concat_columns` and it takes both series and dfs :)

https://woodwork.alteryx.com/en/stable/generated/woodwork.utils.concat_columns.html#woodwork.utils.concat_columns